agent/command/shell.go

```diff
@@ -82,6 +82,10 @@ func (*shellExec) Name() string { return evergreen.ShellExecCommandName }
 
 // ParseParams reads in the command's parameters.
 func (c *shellExec) ParseParams(params map[string]interface{}) error {
+	if params == nil {
```

Comment:
Does ParseParams get called in the validator? Or else can we add something like https://github.com/evergreen-ci/evergreen/blob/ddcfe0c158c9db27872d9735d492d8c3906264b5/validator/project_validator.go#L951 ?

Reply:
ParseParams gets called in the validator - it begins with command.Render at this line and cascades down to call it: https://github.com/evergreen-ci/evergreen/blob/ddcfe0c158c9db27872d9735d492d8c3906264b5/validator/project_validator.go#L932

Reply:
Ah i guess we don't have script in parseParams bc it's just a warning. It would be nice if that could be an error too since idk what the point is without a script (could make that a separate ticket if you prefer)